src/schema/me/followed_fairs.ts

```diff
@@ -0,0 +1,62 @@
+import { ProfileType } from "schema/profile"
+import Fair from "schema/fair"
+import { IDFields } from "schema/object_identification"
+
+import { pageable, getPagingParameters } from "relay-cursor-paging"
+import { connectionDefinitions, connectionFromArraySlice } from "graphql-relay"
+import { GraphQLObjectType } from "graphql"
+
+const FollowedProfileEdge = new GraphQLObjectType({
+  name: "FollowedProfileEdge",
+  fields: {
+    profile: {
+      type: ProfileType,
+    },
+    ...IDFields,
+  },
+})
+
+// the challenge is that FollowedProfile doesn't directly connect to Fair.
+// we need to go from FollowedProfile to Profile to Owner in order to get the
+// reference to the fair object - how do I tell graphQL how to do that?
+// Is it just manually down in my resolver below?
```

Comment:
Does the implementation in [L58](https://github.com/artsy/metaphysics/pull/1441/files#diff-908679437786a6422f3da2c394b44d14R58) satisfy your question here? If so can we remove the comment?